code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -5,26 +5,30 @@
 	desc = "A 12 gauge lead slug."
 	icon_state = "blshell"
 	caliber = "shotgun"
+	var/high_power = TRUE
```

Comment:
This is purely a suggestion and not a forced requirement for merging:

Instead of makeshift/shitty shotguns not being allowed to use high_power rounds, would you consider making them have a chance to explode and harm the user if they try to use high powered rounds?

Reply:
That'd probably be the better solution instead of a hard-ban on high-power shells. I'm not sure if I'd prefer it to be a flat percent chance (probably around 15%) each shot, or if high-power ammo 'damages' the barrel and builds up a malfunction chance (with it guaranteed to explode after 5 or so shots).

Reply:
Apologies but I have to do the cliché and state my unsought opinion. I hate RNG based things like that, I myself would prefer if there was a set number of shots before it blew up. Its more fair that way.

Reply:
Strongly disagree. 

RNG absolutely deserves a place in combat within an RP oriented setting. It would not deserve a place if this were a heavily skill-based pvp server, but that isn't the focus here.

Reply:
I do think RNG in something like this where it only effects a person who gets greedy is far less egregious than say the tonfa baton PR where I said the same thing as that had the chance to just instantly sleep someone you used it on.

Reply:
Just added a ~~bullshit RNG~~ stress mechanic to improvised shotguns. 10% base chance to explode (similar to det's revolver) that slowly goes up as high-power ammo is fired through it (+5% per shot). After enough strain is put on the gun (+30% / 6 shots), there's a slim chance (5%) for low-power ammo to cause a detonation as well. The reinforced variable is used to ignore this mechanic now.

Video is under testing evidence. Feels pretty alright in the little bit of testing I did with it tbh. There are pop-ups at certain stress thresholds, and it gives a vague idea of the current stress level in the examine prompt.

Reply:
It's almost eerie how that matches the compromise I was going to suggest number for number but got interrupted before I could finish typing it on lunch

Reply:
Is this the extremely rare compromise that makes everyone equally happy instead of equally unhappy? I think it might be.